lib/developer_portal/app/controllers/developer_portal/login_controller.rb

```diff
@@ -31,19 +31,29 @@ def create
       self.current_user = @user
       create_user_session!
       flash[:notice] = @strategy.new_user_created? ? 'Signed up successfully' : 'Signed in successfully'
+
+      AuditLogService.call("Signed in: #{current_user.id}/#{current_user.username} #{current_user.first_name} #{current_user.last_name}")
```

Comment:
Isn't full name a little bit too much logging?

Also, provided that we audit UserSession, why do we need a separate log message that a user logged in? Maybe one can watch the sessions creation audits.

Reply:
> Isn't full name a little bit too much logging?

Maybe, I copied this from what we do in the admin portal:

https://github.com/3scale/porta/blob/ca21bb0a1fa8f6dd1a8a4773ad0891f5e8c666b1/app/controllers/provider/sessions_controller.rb#L31

> Also, provided that we audit UserSession, why do we need a separate log message that a user logged in? Maybe one can watch the sessions creation audits.

Maybe it's not required. I added the log because that's what the issue description says. But I'm open to remove this traces if you think they are not needed.

Reply:
And what about the audit log already existing in the admin portal, should we remove it also? It's not required anymore now we are auditing `UserSession`

Reply:
+1, with a note in the release notes.

Reply:
Aternatively we may stop auditing session and keep only the log :sunglasses:

Reply:
Also added some release notes to the issue.